### PR TITLE
Bug fix for Spiral Test

### PR DESCRIPTION
**Pull request by DesAndrew**
Also added my userID resource

### CODE REVIEW

**Comment by jakrach** [comment]
Please don't commit your personal userID resource (keep that local)--everyone has their own.

**Review by krockpot — CHANGES_REQUESTED** [review]
Remove userID file.

File: app/src/main/res/values/userID.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="userID">t10p26</string>
```
Comment (MrPickles) [inline app/src/main/res/values/userID.xml:3]: How was this not blocked by the `.gitignore`? Was branch created before `userID.xml` was put in the `.gitignore`, or was this manually added?

**Review by MrPickles — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_spiral.xml [hunk 2]
```diff
@@ -69,14 +69,15 @@
         android:layout_marginTop="427dp" />
 
     <TextView
-        android:text="@string/spiral_info"
+        android:text="Trace spiral"
```
Comment (MrPickles) [inline app/src/main/res/layout/activity_spiral.xml:72]: Why change this to be a hard coded string instead of a constant from `strings.xml`?

**Pull request closed without merging**